lib/Targets/AIETargets.cpp

```diff
@@ -164,16 +156,16 @@ void registerAIETranslations() {
                 writeBufferMap(output, buf, offset, NL);
           };
 
-          const auto &target_model = xilinx::AIE::getTargetModel(srcTileOp);
+          const auto &targetModel = xilinx::AIE::getTargetModel(srcTileOp);
 
-          if (auto tile = target_model.getMemSouth(srcCoord))
-            doBuffer(tile, target_model.getMemSouthBaseAddress());
-          if (auto tile = target_model.getMemWest(srcCoord))
-            doBuffer(tile, target_model.getMemWestBaseAddress());
-          if (auto tile = target_model.getMemNorth(srcCoord))
-            doBuffer(tile, target_model.getMemNorthBaseAddress());
-          if (auto tile = target_model.getMemEast(srcCoord))
-            doBuffer(tile, target_model.getMemEastBaseAddress());
+          if (auto tile = targetModel.getMemSouth(srcCoord))
+            doBuffer(tile, targetModel.getMemSouthBaseAddress());
+          if (auto tile = targetModel.getMemWest(srcCoord))
+            doBuffer(tile, targetModel.getMemWestBaseAddress());
+          if (auto tile = targetModel.getMemNorth(srcCoord))
+            doBuffer(tile, targetModel.getMemNorthBaseAddress());
+          if (auto tile = targetModel.getMemEast(srcCoord))
+            doBuffer(tile, targetModel.getMemEastBaseAddress());
```

Comment:
These style fixes are not really part of the path, and they would be better on their own, in its own fix, with its own description.

Reply:
It's true but there's so many that it's basically impossible to do that.

Reply:
Why? You can leave them as they are, and do another patch that says "[nfc] change target model variable name to camelcase". Not saying you must do that for this patch right now, but patches are more manageable when they are single issue, rather than "<issue>(+unspecified things here and there)". The "side things" are lost from the history, which may be source of pain further down the line, so it's something to avoid.

Reply:
> Not saying you must do that for this patch right now, but patches are more manageable when they are single issue, rather than "(+unspecified things here and there)". The "side things" are lost from the history, which may be source of pain further down the line, so it's something to avoid.

Of course I agree and I would rather not have to do this (and since you prefer I didn't, I won't anymore) but here's what I'm saying

1. These kinds of lint are absolutely everywhere;
2. They are hard to find (a `[a-z]_[a-z]` regex catches stl methods and such);
3. They're not actually a priority.

So short of just keeping a list of line numbers, I don't see how to aggregate these into one NFC. So I'm taking the approach [recommended here](https://github.com/Xilinx/mlir-air/pull/286#pullrequestreview-1594655052).